src/include/fs_types.h

```diff
@@ -4,6 +4,7 @@
 #define CEPH_INCLUDE_FS_TYPES_H
 
 #include "types.h"
+#include "common/ceph_json.h"
```

Comment:
maybe just declear JSONObj and include ceph_json.h in cc files

Reply:
changed